lib/config.js

```diff
@@ -48,6 +48,7 @@ config.cronIntervalCalculateAssessmentQuestionStatsSec = 10 * 60;
 config.cronDailySec = 8 * 60 * 60;
 config.autoFinishAgeMins = 6 * 60;
 config.questionDefaultsDir = 'question-servers/default-calculation';
+config.questionTimeout = 10000;
```

Comment:
```suggestion
config.questionTimeoutMilliseconds = 10000;
```

We typically put the unit on time-related config variables, because they are variously in milliseconds, seconds, minutes, etc, and it's really annoying to have to go and read the code to figure out what the units are 😄 

We were using "MS" for "Milliseconds" on other config variables, but I actually think we should just write it out for clarity.